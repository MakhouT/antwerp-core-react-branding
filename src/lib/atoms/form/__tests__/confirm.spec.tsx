import * as React from "react";

import * as A from "../../..";
import { renderAndMatchSnapshot } from "../../../../__tests__/__fixtures__/testUtils";

describe("Confirm", () => {
    it("Renders default", () => {
        // Arrange
        const element = <A.Confirm
            name="confirm"
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
