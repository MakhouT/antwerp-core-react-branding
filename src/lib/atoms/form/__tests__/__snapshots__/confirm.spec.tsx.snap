// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm Renders default 1`] = `
<div
  className="a-input"
>
  <div
    className="a-input__checkbox"
  >
    <input
      checked={false}
      disabled={undefined}
      id="checkbox-confirm-0"
      name="confirm"
      onChange={[Function]}
      type="checkbox"
      value="true"
    />
  </div>
  <small
    className=""
  >
     
  </small>
</div>
`;
